region/lowernorfair/east/Mickey Mouse Room.json

```diff
@@ -1423,137 +1396,117 @@
-              "Multiviola"
-            ]
-          ],
-          "explicitWeapons": ["Missile"]
+          "enemies": [["Dessgeega", "Dessgeega"]],
+          "explicitWeapons": ["Super"]
         }},
-        {"heatFrames": 480}
+        {"heatFrames": 200}
       ],
-      "clearsObstacles": ["C", "E"]
+      "clearsObstacles": ["F"]
     },
     {
-      "link": [8, 7],
-      "name": "Missiles Kill Sparing Multiviolas",
+      "link": [7, 6],
+      "name": "Screw Kill",
       "requires": [
-        "h_canNavigateHeatRooms",
-        {"enemyKill": {
-          "enemies": [["Dessgeega", "Dessgeega", "Dessgeega"]],
-          "explicitWeapons": ["Missile"]
-        }},
-        {"heatFrames": 540}
+        "ScrewAttack",
+        {"heatFrames": 160}
       ],
-      "clearsObstacles": ["C"],
-      "note": [
-        "Sparing the Multiviolas is required to perform an ice clip further in.",
-        "However, it requires going slower."
-      ]
+      "clearsObstacles": ["F"]
     },
     {
-      "link": [8, 7],
-      "name": "Supers Kill",
+      "link": [7, 6],
+      "name": "Missiles Kill",
       "requires": [
-        "h_canNavigateHeatRooms",
+        {"obstaclesCleared": ["C"]},
         {"enemyKill": {
-          "enemies": [
-            [
-              "Dessgeega",
-              "Dessgeega",
-              "Dessgeega",
-              "Multiviola",
-              "Multiviola"
-            ]
-          ],
-          "explicitWeapons": ["Super"]
+          "enemies": [["Dessgeega", "Dessgeega"]],
+          "explicitWeapons": ["Missile"]
         }},
-        {"heatFrames": 380}
+        {"heatFrames": 400}
       ],
-      "clearsObstacles": ["C", "E"]
+      "clearsObstacles": ["F"]
     },
     {
-      "link": [8, 7],
-      "name": "Supers Kill Sparing Multiviolas",
+      "link": [8, 1],
+      "name": "Get Hit on Entry",
       "requires": [
-        "h_canNavigateHeatRooms",
-        {"enemyKill": {
-          "enemies": [["Dessgeega", "Dessgeega", "Dessgeega"]],
-          "explicitWeapons": ["Super"]
-        }},
-        {"heatFrames": 380}
-      ],
-      "clearsObstacles": ["C"],
-      "note": [
-        "Sparing the Multiviolas is required to perform an ice clip further in.",
-        "However, it requires going slower."
+        {"enemyDamage": {
+          "enemy": "Multiviola",
+          "type": "contact",
+          "hits": 1
+        }}
       ]
     },
     {
-      "link": [8, 7],
-      "name": "Screw Kill",
+      "link": [8, 1],
+      "name": "Kill Multiviola on Entry",
       "requires": [
-        "h_canNavigateHeatRooms",
-        "ScrewAttack",
-        {"heatFrames": 230}
+        "canPrepareForNextRoom",
+        {"or": [
+          {"and": [
+            {"enemyKill": {
+              "enemies": [["Multiviola"]],
+              "explicitWeapons": ["PseudoScrew", "ScrewAttack"]
+            }},
+            {"heatFrames": 30}
+          ]},
+          {"and": [
+            "canTrickyJump",
```

Comment:
Unfortunately this PR is very hard to read because of moving strats. I didn't change much besides updating a few dev/notes. I did add this tricky jump requirement for missiles/supers